src/main/java/com/NewsWorkflw/common/controller/RssController.java

```diff
@@ -36,6 +38,14 @@ public ModelAndView getFeedInRss() {
 		mav.setViewName("rssViewer");
 		mav.addObject("feedContent", items);
 		
+		ModelAndView modelview = new ModelAndView();
+		modelview.setViewName("rssViewer");
+		modelview.addObject("feedContent"), items);
+		
+		//renamed mav to modelview, makes it more explainable

```

Comment:
Save this discussion for the pull request (where I am putting this comment; you can have inline conversations here).  It should not be in comments in the source code, because that's clutter, which is technical debt.